packages/react/src/state.tsx

```diff
@@ -115,4 +134,43 @@ export const increment = (value: number): IncrementStateModifier => ({
   value,
 })
 
-export const LocalState = createContext<LocalState>({})
+export const useLocalStateValues = () => {
+  const localState = useContext(LocalState)
+
+  const [, forceUpdate] = useState({})
+
+  const streams = useRef(new Map<string, Stream<JSONValue>>())
```

Comment:
if the state dependencies change, I guess this does not ever clean up the unused streams?

Because `get` for all states should happen during every render, maybe (maybe!) it makes sense to do this:
```
  const streams = useRef(new Map<string, Stream<JSONValue>>())
  streams.current = new Map()
```

Reply:
Going to turn this into an issue to address later. Now that I think about it, it's unlikely state dependencies change (at least within kit package), so while it's possible to support this, I can't think of a nice and easy unit test case to test it.